src/components/autonomous/WaypointNav.js

```diff
@@ -18,6 +20,17 @@ function WaypointNav() {
     dispatch(requestWaypointNav(formJson));
   };
 
+  function grabFromClipboard () {
+    navigator.clipboard.readText().then(text => {
+      if(text.match("-?\\d{1,}\\.?\\d{0,}, -?\\d{1,}\\.?\\d{0,}")) {
+        setLat(text.split(", ")[0]);
+        setLon(text.split(", ")[1]);
+      }
```

Comment:
Could we add an `else` that prints to the console or shows an alert (maybe both?) if the text on the clipboard doesn't match the regex? Failing silently might confuse users.